streamer/parser/a11y-metadata-parsing.md

```diff
@@ -0,0 +1,271 @@
+# Parsing Accessibility Metadata
+
+The goal of this document is to provide directions that each implementation of Readium can follow when parsing schema.org Accessibility Metadata in EPUB publications.
+
+When parsing a publication in the streamer we always use the most complex form for each metadata to harmonize our output.
+
+Related Repository: [Readium Web Publication Manifest](https://github.com/readium/webpub-manifest)
+
+References: 
+
+- http://kb.daisy.org/publishing/docs/metadata/schema-org.html
+- http://kb.daisy.org/publishing/docs/metadata/evaluation.html
+- https://www.w3.org/wiki/WebSchemas/Accessibility
+
+## AccessMode
+
+`accessMode` is a key whose value is an array of strings.
+
+It is expected in a conformant EPUB publication ([EPUB Accessibility 1.0](https://www.w3.org/Submission/epub-a11y/#sec-disc-package)).
+
+At the time of writing, known values are:
+
+- `auditory`
+- `tactile`
+- `textual`
+- `visual`
+- `chartOnVisual`
+- `chemOnVisual`
+- `colorDependent`
+- `diagramOnVisual`
+- `mathOnVisual`
+- `musicOnVisual`
+- `textOnVisual`
+
+### EPUB 2.x
+
+The array is created from the `meta` elements whose `name` attribute has the value `schema:accessMode`. The value of their `content` attribute is pushed to the array.
+
+### EPUB 3.X
+
+The array is created from the `meta` elements whose `property` attribute has the value `schema:accessMode`. Their value is pushed to the array.
```

Comment:
> Their value is pushed to the array.

Maybe to clarify:

"Their value (i.e. child textual content) is pushed to the array."

or:

"Their value (i.e. child text node) is pushed to the array."

(I have a preference for the former, as it is not technically-bound to the notion of unique child node)